ci/libvcx.dockerfile

```diff
@@ -1,81 +1,76 @@
-FROM base:1.0.0 AS BASE
+FROM alpine:3.12 AS builder
 
-# Install Rust toolchain
-ARG RUST_VER=1.40.0
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain ${RUST_VER}
-ENV PATH /home/indy/.cargo/bin:$PATH
+ARG UID=1000
+ARG GID=1000
 
-# Clone indy-sdk
+ARG INDYSDK_PATH=/home/indy/indy-sdk
+ARG INDYSDK_REPO=https://github.com/hyperledger/indy-sdk.git
 ARG INDYSDK_REVISION=v1.15.0
-ARG INDYSDK_REPO=https://github.com/hyperledger/indy-sdk
-WORKDIR /home/indy
-RUN git clone "${INDYSDK_REPO}" "./indy-sdk"
-RUN cd "/home/indy/indy-sdk" && git checkout "${INDYSDK_REVISION}"
-
-# Build indy binaries and move to system library
-# TODO: It should be possible to run a single command as a user
-RUN cargo build --release --manifest-path=/home/indy/indy-sdk/libindy/Cargo.toml
-USER root
-RUN mv /home/indy/indy-sdk/libindy/target/release/*.so /usr/lib
-USER indy
-RUN cargo build --release --manifest-path=/home/indy/indy-sdk/libnullpay/Cargo.toml
-RUN cargo build --release --manifest-path=/home/indy/indy-sdk/experimental/plugins/postgres_storage/Cargo.toml
-USER root
-RUN mv /home/indy/indy-sdk/libnullpay/target/release/*.so /usr/lib
-RUN mv /home/indy/indy-sdk/experimental/plugins/postgres_storage/target/release/*.so /usr/lib
 
-# Build libvcx
-WORKDIR /home/indy
-USER indy
-COPY --chown=indy  ./ ./
-RUN cargo build --release --manifest-path=/home/indy/libvcx/Cargo.toml
-
-# Move the binary to system library
-USER root
-RUN mv /home/indy/libvcx/target/release/*.so /usr/lib
+ENV RUST_LOG=warning
 
+RUN addgroup -g $GID indy && adduser -u $UID -D -G indy indy
 
-# Create a new build stage and copy outputs from BASE
-FROM ubuntu:16.04
-
-RUN apt-get update && \
-    apt-get install -y \
-      libssl-dev \
-      apt-transport-https \
-      ca-certificates \
-      curl \
-      build-essential
-
-RUN useradd -ms /bin/bash -u 1000 indy
+RUN apk update && apk upgrade && \
+    apk add --no-cache \
+        build-base \
+        cargo \
```

Comment:
Is it possible to install specific version of cargo / rust toolchain (passed as a build ARG)? If not, we should also remove the unnecessary `--build-arg` flags from CI.